File: python/ZoneVerifier.py
import os

def banner():
    os.system('cls')
    print("=========================")
    print("===== Zone Verifier =====")
    print("=========================")

banner()

print("Este programa irá ler o código do produto que você inserir, e iremos informar de qual região do Brasil o produto vem, ou se ele é importado.")

codigo_produto = float(input("Insira o código do produto, localizado na embalagem do mesmo: "))


match codigo_produto:
    case 1:
        regiao = "Este produto é da região Sul."
    case 2:
        regiao = "Este produto é da região Norte."
    case 3:
        regiao = "Este produto é da região Leste."
    case 4:
        regiao = "Este produto é da região Oeste."
    case 5:
        regiao = "Este produto é da região Nordeste."
    case 6:
        regiao = "Este produto é da região Nordeste."
    case 7:
        regiao = "Este produto é da região Sudeste."
    case 8:
        regiao = "Este produto é da região Sudeste."
    case 9:
        regiao = "Este produto é da região Sudeste."
    case 10:
        regiao = "Este produto é da região Centro-Oeste."
    case 11:
        regiao = "Este produto é da região Noroeste."
    case _:
        regiao = "Este produto é IMPORTADO!!!"

os.system('cls')

banner()

print(regiao)

File: c/ZoneVerifier.c
#include <stdio.h>
#include <stdlib.h>
#include <windows.h>
#include <string.h>

int banner() {
    system("cls");
    printf("|=========================|\n");
    printf("|===== Zone Verifier =====|\n");
    printf("|=========================|\n");
}

int main(){
    banner();

    float codigo_produto;
    char regiao[50];

    printf("Esse programa ira ler o codigo do produto que voce inserir, e iremos informar de qual regiao do Brasil o produto vem, ou se ele e importado.\n");

    printf("Insira o codigo do produto, localizado na embalagem do mesmo: ");
    scanf("%f" ,&codigo_produto);

    switch ((int)codigo_produto) {
        case 1: strcpy(regiao, "Este produto e da regiao Sul.");
         break;
        case 2: strcpy(regiao, "Este produto e da regiao Norte");
         break;
        case 3: strcpy(regiao, "Este produto e da regiao Leste");
         break;
        case 4: strcpy(regiao, "Este produto e da regiao Oeste");
         break;
        case 5: strcpy(regiao, "Este produto e da regiao Nordeste");
         break;
        case 6: strcpy(regiao, "Este produto e da regiao Nordeste");
         break;
        case 7: strcpy(regiao, "Este produto e da regiao Sudeste");
         break;
        case 8: strcpy(regiao, "Este produto e da regiao Sudeste");
         break;
        case 9: strcpy(regiao, "Este produto e da regiao Sudeste");
         break;
        case 10: strcpy(regiao, "Este produto e da regiao Centro-Oeste");
            default:
                strcpy(regiao, "Este produto é IMPORTADO!!!");
                break;
    }

    system("cls");

    printf("\n%s", regiao,"\n");
}
